Cannot build 'mirage-dev' website with 'mirageos-3-beta remote'.

I just tried to build the `mirage-dev` branch of the site, following the instructions in the relevant markdown file from that branch.  I got the following:

```
$ pwd
/Users/amir/repos/mirage/mirage-www/src
$ mirage configure -t unix --kv_ro crunch --net socket
$ make
mirage build

run ['ocaml-crunch' '-o' 'static1.ml' '../files']: exited with 127make: *** [build] Error 1
```

Some additional info:
```
$ opam config report
# OPAM config report
# opam-version    1.2.2
# self-upgrade    no
# os              darwin
# external-solver aspcud $in $out $criteria
# criteria        -count(removed),-notuptodate(request),-sum(request,version-lag),-count(down),-notuptodate(changed),-count(changed),-notuptodate(solution),-sum(solution,version-lag)
# jobs            4
# repositories    1* (http), 1 (version-controlled)
# pinned          0
# current-switch  mirage3*
# last-update     2017-02-22 22:59
$ opam remote
  10 [git]     mir-beta     https://github.com/mirage/mirageos-3-beta.git
   0 [http]     default     https://opam.ocaml.org
```

Am I doing something wrong?

--- a/tmpl/wiki/mirage-www.md
+++ b/tmpl/wiki/mirage-www.md
@@ -53,6 +53,7 @@ Then configure and build the website itself:
 ```
 $ cd src
 $ mirage configure -t unix --kv_ro crunch --net socket
+$ make depend
 $ make
 ```
 

